bundle/regal/ast.rego

```diff
@@ -191,3 +191,50 @@ implicit_boolean_assignment(rule) if {
 all_functions := object.union(opa.builtins, function_decls(input.rules))
 
 all_function_names := object.keys(all_functions)
+
+comments_decoded := [decoded |
+	some comment in input.comments
+	decoded := object.union(comment, {"Text": base64.decode(comment.Text)})
+]
+
+comments["blocks"] := comment_blocks(comments_decoded)
+
+comments["metadata_attributes"] := {
+	"scope",
+	"title",
+	"description",
+	"related_resources",
+	"authors",
+	"organizations",
+	"schemas",
+	"entrypoint",
+	"custom",
+}
+
```

Comment:
This could become a bit of a hassle if OPA introduces further attributes. But OK for now.

Reply:
True. Not sure if there's a way to enumerate them from the Go API, but that would be preferable 🤔